Headlines/src/Categories/UIImageEffects.swift

```diff
@@ -0,0 +1,281 @@
+/*
+ File: UIImage+ImageEffects.m
+ Abstract: This is a category of UIImage that adds methods to apply blur and tint effects to an image. This is the code you’ll want to look out to find out how to use vImage to efficiently calculate a blur.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/b49c86a9b05bec3bbe17833b5a716a8293c42cf1/Headlines/src/Categories/UIImageEffects.swift#L3"></span>Line should be 120 characters or less: currently 206 characters</td>
    </tr>
  </tbody>
</table>